Organic Impressions Estimator/Code.js

```diff
@@ -79,65 +79,6 @@ function fetchRankingData() {
   getAllRankingData(initialUrl, options, primaryAsin, competitorAsins, rankedKeywordsOnly, minMonthlySearchVolume, newRankingData);
 }
 
-/**
- * Collects all the data from the "Raw Rank Data" sheet and populates the "Rank by Day" sheet.
- */
-function populateRankByDaySheet() {
```

Comment:
I think I used this in the function `getRanksAndPopulateRankByDay` for the `runAllSheets` function --> if you run:

```
git rebase main
git push --force 
```

It'll pull those changes in with yours laid on top of them, then you can go back and delete any other usage of this function.

Reply:
`  fetchRankingData` calls `getAllRankingData` which calls `populateRankByDaySheet`.  I don't think we need `getRanksAndPopulateRankByDay` for this reason. Do you suggest we structure it differently?

Reply:
I added `populateRankByDaySheet` back in and changed `runAllSheets` to run the correct functions.